recipe/meta.yaml

```diff
@@ -20,19 +20,21 @@ source:
     - osx.patch  # [osx]
 
 build:
-  number: 0
+  number: 1
   detect_binary_files_with_prefix: true
   skip: true  # [win]
 
 requirements:
   build:
     - gcc  # [osx]
+    - libgcc  # [osx]
```

Comment:
Well, if you look at the build logs, the attempted fix didn't work — the resulting dylib still has a zany path for `libgcc_s` in the end. I'll need to do some more experimentation.

That being said, I think the `gcc` package is broken since it apparently contains a hardcoded path inside `/Users/ray` that shouldn't make it into the distributed package.